stencil/src/components/pulumi-api-symbol/pulumi-api-symbol.scss

```diff
@@ -0,0 +1,26 @@
+.symbol {
+  height: 24px;
```

Comment:
does this need to be set, only to be overriden by the large/small class height + widths?